### PR TITLE
DRAFT: Proposal for translation

### PR DESCRIPTION
**Pull request by larowlan**

### CODE REVIEW

File: tests/Infrastructure/Aggregate/PreconditionsTree/BeginnerPreconditionsUnitTest.php [hunk 1]
```diff
@@ -52,13 +52,13 @@ public function testFulfilled(): void
         $stagingDir = $this->stagingDir->reveal();
         $exclusions = $this->exclusions;
         $this->commonPreconditions
-            ->assertIsFulfilled($activeDir, $stagingDir, $exclusions)
+            ->assertIsFulfilled($activeDir, $stagingDir, $exclusions, $exclusions)
```
Comment (larowlan) [inline tests/Infrastructure/Aggregate/PreconditionsTree/BeginnerPreconditionsUnitTest.php:55]: ignore all these, phpstorm refactoring helpfully just dupes the argument if it has no idea.
if there's broad support for the idea I'll fix all these

**Comment by TravisCarden** [comment]
Can you give a rough idea what an implementation of `TranslationInterface` might look like in this proposal, @larowlan?

**Comment by effulgentsia** [comment]
@larowlan: With this approach, how do you envision localize.drupal.org finding the translatable strings during static analysis?

**Comment by TravisCarden** [comment]
Closing based on discussion in https://github.com/php-tuf/composer-stager/issues/123, which has taken a different direction.

**Pull request closed without merging**